src/Factory/ActivityPub/EntryPageFactory.php

```diff
@@ -18,6 +18,16 @@
 
 class EntryPageFactory
 {
+    public const ADDITIONAL_CONTEXTS = [
+        'lemmy' => 'https://join-lemmy.org/ns#',
```

Comment:
I think https://socialhub.activitypub.rocks/t/we-have-created-documentation-on-how-exactly-lemmy-federation-works/1085/20?u=trwnh probably has the information I was looking for

Reply:
yeah it's basically what's in that link - some of properties mbin emit in AP responses right now could not be expanded into its full form under json-ld, and this change is for adding back those missing contexts and property mappings so it could be expanded properly under json-ld (try `curl -vL -H 'Accept: application/activity+json, application/ld+json' $url` on some of our objects/actors, chuck them into [json-ld playground](https://json-ld.org/playground/), and click for "canonized" visualization, quite a bit of them would results in an error on top on output box)

and I think it's less of wanting to advertise other software, but rather they have made property extensions with matching semantic for both them and us, so we're using them for proper communication, we could totally create our own extension if needed/wanted (and I kind of want/plan to do this when it's time for badge federation)